src/styles/card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  transition-duration: 300ms;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__info {
+    font-family: $font;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $mainColor;
+    padding-bottom: 4px;
+  }
+
+  &__productCode {
+    font-family: $font;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: $fontWeightStandard;
+    padding-bottom: 16px;
+    color: $secondColor;
+  }
+
+  &__buyButton {
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    border: none;
+    color: #fff;
+    background-color: #00acdc;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+
+    cursor: pointer;
+
+    &:hover {
+      transition-duration: inherit;
```

Comment:
The 'transition-duration' property inside the ':hover' selector is unnecessary since it inherits the value from the base class. You can safely remove this line.